### ISSUE TITLE
JAWS reading old input value in browser mode while dragging input

### ISSUE THREAD

**Issue by venkitp** [FreedomScientific/VFO-standards-support#886]
I'm using an range input with label mapped inside a wrapped div. When i enable browse mode and drag the input slider and focus on the thumb using browser mode (on up and down arrow keys). It reads out the previous value of the input. This happens in subsequent changing of values in the input. 

Happens only when dragging the slider and not clicking to change the value.

Please help in getting a solution. Need either a workaround or fix.

Code used

<div>
    <label for="slider">label</label>
    <input type="range" id="slider" name="slider">
</div>

**Comment by JAWS-test** [comment]
Your code is not visible, so the problem is difficult to understand. 

But: if I use an input type=range, I only hear the value correctly with JAWS when using arrow keys for changing the value. If I operate the element with the mouse (whether by click or by drag & drop), JAWS does not output anything.

**Comment by JAWS-test** [comment]
If JAWS is used by blind people, then the output only during arrow key operation is OK, as they cannot operate the element otherwise. The lack of output with mouse operation is problematic for visually impaired people who may want to operate the element with the mouse.

**Comment by JeremiahRogersVispero** [comment]
Using the code, <label for="vol">Volume:</label><input id="vol" max="100" min="0" name="vol" type="range" />, could not reproduce the issue as described
